setup.py

```diff
@@ -99,7 +102,26 @@ def gen_packages_items():
     return packages
 
 
+def link_file_to_package():
+    """Create symlinks to package dependencies to support MIM."""
+    item_list = ['tools', 'configs', 'demo', 'model_zoo.yml']
```

Comment:
model_zoo.yml has been renamed to [model-index.yml](https://github.com/open-mmlab/mmpose/blob/master/model-index.yml), so the 'model_zoo.yml' should be renamed to 'model-index.yml'